File: hsPlayerProfile/webServices/src/webServices/Entities/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace webServices.Entities
{
    public class Student : IEntityBase
    {
        public Student()
        {
        }

        public int id { get; set; }
        public string lastName { get; set; }
        public string firstName { get; set; }
        public int highSchoolId { get; set; }
        public string profileName { get; set; }
        public string primaryEmail { get; set; }
    }
}

File: hsPlayerProfile/web/app/models/Student.ts
import { IStudent } from './IStudent';

export class Student implements IStudent {
	public id: number;
    public profileName: string;
    public firstName: string;
    public lastName: string;
    public primaryEmail: string;
    public highSchoolId: number;
    public graduationYear: number;
    public additionalInfo: string;
    public gpa: number;
    public satTestDate: Date;
    public satScore: number;
    public actTestDate: Date;
    public actScore: number;
    public ncaaid: string;
    public phone: number;
    public streetAddress1: string;
    public streetAddress2: string;
    public city: string;
    public state: string;
    public zip: string;
    public displayAddrAndPhone: number;
}